package.json

```diff
@@ -5,7 +5,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "prettier-format": "prettier --config .prettierrc.json '**/*.tsx' --write"
```

Comment:
we want `prettier` to format more than just the `tsx` files, so I reckon change the command to `prettier --config .prettierrc.json ./ --write` and have a `.prettierignore` file as following

```
node_modules
.next
out
```

When run `yarn prettier-format` it would format the entire codebase except those folders listed above.